composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "laravel/laravel",
+    "name": "adshares/adserver",
     "description": "The Laravel Framework.",
```

Comment:
Maybe we can change also description? :)

Reply:
:+1: